app/models/paper.rb

```diff
@@ -10,7 +10,7 @@ class Paper < ApplicationRecord
   has_many :links, dependent: :destroy
   has_many :api_import_responses, dependent: :destroy
 
-  accepts_nested_attributes_for :authors, reject_if: proc { |attributes| attributes['name'].blank? }
+  accepts_nested_attributes_for :authors, reject_if: proc { |attributes| attributes['last_name'].blank? }
```

Comment:
I kept it first and last name because it literally refers to the positionality of the names. Also, I'm not sure it matters what we call it if we just have two parts, unless we're talking about how we display it on the frontend.